lib/loadEnv.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const dotenv = require('dotenv');
+const path = require('path');
+const ServerlessError = require('./classes/Error').ServerlessError;
+
+const isMissingFileError = error => error.code === 'ENOENT';
+
+const throwDotenvError = (error, filePath) => {
+  const errorMessage = [
+    `Encountered: "${error}" while trying to load environment variables`,
+    ` from filepath: ${filePath}.`,
```

Comment:
I'd probably reword it to:

```
Failed to load environment variables from "${filePath}": ${error}"
```

- Original error message can be verbose, so it's better to put this at the end
- It's also convention in few other places, to first provide short context info, and then trail it with original error message.